### PR TITLE
Hotfix for build error when compiled with '--clients-only' option 

### LINKED ISSUE #1241: rocblas-clients  build error for ROCm 5.1.1
### What is the expected behavior
- I want to only build rocblas clients for ROCm 5.1.1 inside docker image [rocm/pytorch:rocm5.1.1_ubuntu20.04_py3.7_pytorch_1.10.0](https://registry.hub.docker.com/layers/pytorch/rocm/pytorch/rocm5.1.1_ubuntu20.04_py3.7_pytorch_1.10.0/images/sha256-df3e74b4f80ba64ae3d040824ee49e858c5ee4810f44a3747e744b577a928b24?context=explore)

### What actually happens
- It failed to build and I got the 'header file tensile_host.hpp not found in rocBLAS/clients/benchmarks/client.cpp ' error 
image

### How to reproduce
git clone -b rocm-5.1.1 https://github.com/ROCmSoftwarePlatform/rocBLAS.git ./rocBLAS
cd ./rocBLAS && ./install.sh --dependencies --clients-only 

I can fix this issue by 
 - adding '    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../../library/src/include>' in rocBLAS/clients/benchmarks/CMakeLists.txt at line#20 
**or** 
 - change the tensile_host.hpp to right relative path in rocBLAS/clients/benchmarks/client.cpp
 
but could you please help me on how to add this patch into ROCm 5.1.1 branch or tag?

### PR DESCRIPTION
**Pull request by rkamd**
resolves #1241 

Summary of proposed changes:
- Updated internal includes list for clients directory

### CODE REVIEW

**Review by TorreZuk — APPROVED** [review]
LGTM if all tests pass, please verify git commands in CI are what would expect.  Await external approval before merging as discussed already.

**Review by amcamd — APPROVED** [review]

**Pull request merged**
